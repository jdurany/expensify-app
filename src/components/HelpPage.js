import React from 'react';

const HelpPage = () => (
  <div>
    This is from my expense help component
  </div>
);

export default HelpPage;