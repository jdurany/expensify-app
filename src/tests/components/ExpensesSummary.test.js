import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should render ExpensesSummary with 1 expense', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235} />);
  expect(wrapper).toMatchSnapshot();
});

test('should render ExpensesSummary with multiple expenses message', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={4} expensesTotal={1234} />);
  expect(wrapper).toMatchSnapshot();
});