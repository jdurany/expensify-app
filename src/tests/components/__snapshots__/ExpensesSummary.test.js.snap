// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseList with empty message 1`] = `
<div>
  <p>
    Viewing 
     expenses totalling 
    $0.00
  </p>
</div>
`;

exports[`should render ExpenseList with expenses 1`] = `
<div>
  <p>
    Viewing 
     expenses totalling 
    $0.00
  </p>
</div>
`;
